apis/externalsecrets/v1beta1/secretstore_senhasegura_types.go

```diff
@@ -0,0 +1,67 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import esmeta "github.com/external-secrets/external-secrets/apis/meta/v1"
+
+/*
+	SenhaseguraAuth tells the controller how to do auth in senhasegura
+*/
+type SenhaseguraAuth struct {
+	IsoSecretRef *SenhaseguraIsoAuth `json:"isoSecretRef"`
+}
+
+/*
+	SenhaseguraIsoAuth holds secrets references for senhasegura authentication parameters
+*/
+type SenhaseguraIsoAuth struct {
+	/* The URL is used for authentication */
+	URL esmeta.SecretKeySelector `json:"urlSecretRef"`
```

Comment:
The URL does not need to be in the Secret, does it?

Reply:
Moved URL and Client **ID** out of secret, [Cluster]SecretStore become more human readable/friendly, and it makes more sense now